chromium_src/components/prefs/pref_service.h

```diff
@@ -0,0 +1,17 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_COMPONENTS_PREFS_PREF_SERVICE_H_
+#define BRAVE_CHROMIUM_SRC_COMPONENTS_PREFS_PREF_SERVICE_H_
+
+#define GetBoolean                                         \
+  GetBooleanOr(const std::string& path, bool other) const; \
```

Comment:
this change is pretty specific and is used only for some specific patches. I think having local functions where they are required to patch upstream logic is a more preferred approach here.